compiler/rustc_hir_analysis/src/errors.rs

```diff
@@ -320,7 +320,8 @@ pub struct AutoDerefReachedRecursionLimit<'a> {
 #[diag(hir_analysis_track_caller_on_main)]
 pub(crate) struct TrackCallerOnMain {
     #[primary_span]
+    #[suggestion(applicability = "maybe-incorrect", code = "")]
     pub span: Span,
-    #[label]
+    #[label(hir_analysis_track_caller_on_main)]
```

Comment:
I think this can be turned back into just `[label]`.

Then you can just add  ".label = `main` function is not allowed to be `#[track_caller]`" back to the ftl file.

Reply:
I did it so because it's the exact same text, so did not want to repeat it.